cafebabel/templates/articles/translations/edit.html

```diff
@@ -1,18 +1,38 @@
 {% extends "base.html" %}
+{% import "articles/macros.html" as macros %}
 {% from "_extrajs-macros.html" import extra_js %}
 
 {% block extra_js %}
-  {{ extra_js(['js/articles-tags.js', 'js/textarea-change.js']) }}
+  {{ extra_js(['js/articles-tags.js', 'js/textarea-change.js', 'js/noise.js']) }}
```

Comment:
I can't find where noise is used in this code. Any hint @cafebabel-webmaster?

Reply:
Agreed, no cover image modification is allowed from that page.

Reply:
true, user can't change cover image at this point. Then I delete it